plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -162,6 +162,7 @@ public void testSortItemsReflectedInExplain()
                 "TopNPartial\\[5 by \\(nationkey DESC");
     }
 
+    @Test
```

Comment:
Was it causing any issues? Maybe we should drop other `@Test` if they are not needed instead?

Reply:
The `@Test` annotation is still needed in the super class for methods that don't override this method. 

I added this annotation here because otherwise it causes issues if another class extending this test suite uses JUnit5 and has to parse testng test methods.